tests/specs/close-db.js

```diff
@@ -5,10 +5,17 @@
         var indexedDB = window.indexedDB || window.webkitIndexedDB || window.mozIndexedDB || window.oIndexedDB || window.msIndexedDB;
 
         beforeEach(function (done) {
-            this.dbName = guid();
+            var spec = this;
+            spec.server = undefined;
+            if (!this.dbName) {
+                this.dbName = guid();
+                done();
```

Comment:
Not true - I want a new db name generated in each test, that way we don't have to worry about the previous db being removed, that way failing tests don't cause a chain reaction and make all following tests fail.